### PR TITLE
Feat: extract message from one guild

### PR DESCRIPTION
**Pull request by magicpaIm**
1. Extract messages from one guild
npm start -- --guild=939491779110117407 (extract from one guild)
npm start (extract from all)
2. Feature to send dm to user

### CODE REVIEW

File: database/dbservice.js [hunk 1]
```diff
@@ -109,9 +109,9 @@ const insertMessages = async (guildID, messages) => {
   return countNewMessages;
 };
 
-// fetch different guild settings from RnDAO server
-const fetchSettings = async () => {
-  const settings = await guildService.fetchGuild();
+// fetch one or different guild settings from RnDAO server
+const fetchSettings = async (guildId) => {
+  const settings = await guildService.fetchGuild(guildId);
```
Comment (mehrdadmms) [inline database/dbservice.js:114]: What if we want to fetch all ?

Comment (magicpaIm) [reply database/dbservice.js:114]: if guildId is null, then it fetchs all settings.
I will update the comment on my code and push again.

File: action/export.js [hunk 2]
```diff
@@ -94,6 +94,31 @@ const fetchMessages = async (
...
+      .catch((e) => {
+        throw e;
+      });
+  } catch (e) {
+    console.log("Can't DM to user", e);
+  }
+};
 module.exports = {
   fetchMessages,
+  sendDMtoUser,
```
Comment (mehrdadmms) [inline action/export.js:123]: You'll need to move this to the server repo.
This is going to be our flow:
- Server will call the bot to extract information
- When extraction is done, server will call the analysis script to do the analysis
- After that we'll notify the user, you may also just import and call it in the server side. it's up to you.

File: index.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ const {
   connectDB,
   updateGuild,
 } = require("./database/dbservice.js");
-const { fetchMessages } = require("./action/export.js");
+const { fetchMessages, sendDMtoUser } = require("./action/export.js");
```
Comment (mehrdadmms) [inline index.js:10]:
```suggestion
const { fetchMessages } = require("./action/export.js");
```

**Comment by mehrdadmms** [comment]
Please add this to readme 

> Extract messages from one guild
npm start -- --guild=939491779110117407 (extract from one guild)
npm start (extract from all)

**Pull request merged**